### ISSUE TITLE
subroutine df_write_unformatted is misnamed

### ISSUE THREAD

**Issue by Beliavsky** [jaiken17/fortranDF#9]
It does a formatted write.

**Comment by jaiken17** [comment]
Yes, it is actually a formatted write, however, I think I named it that way with the idea of adding another write subroutine that lets the user choose the formats. In that sense it is 'unformatted' but a better name would be something like `df_write_default_format`. 

I am planning to overhaul the IO functionality, so I think I will leave it as is until then.
